{"0.1.1":{"info":{"author":"有大家科技有限公司","author_email":"dev@funji.club","bugtrack_url":null,"classifiers":["Development Status :: 4 - Beta","Environment :: Web Environment","Intended Audience :: Developers","License :: OSI Approved :: BSD License","Operating System :: OS Independent","Programming Language :: Python :: 3.9"],"description_content_type":"text/markdown","docs_url":null,"download_url":"","downloads":{"last_day":-1,"last_month":-1,"last_week":-1},"dynamic":null,"home_page":"https://gitlab.corp.funji.club/funji_dev/im/yoto_exception.git","keywords":"","license":"BSD","maintainer":"","maintainer_email":"","name":"yoto-exception","package_url":"https://pypi.org/project/yoto-exception/","platform":"","project_url":"https://pypi.org/project/yoto-exception/","project_urls":{"Homepage":"https://gitlab.corp.funji.club/funji_dev/im/yoto_exception.git"},"provides_extra":null,"release_url":"https://pypi.org/project/yoto-exception/0.1.1/","requires_dist":null,"requires_python":">=3.9","summary":"yoto_api的异常模块","version":"0.1.1","yanked":false,"yanked_reason":null},"last_serial":8669448,"urls":[{"comment_text":"","digests":{"blake2b_256":"f55e17e2e180b9f99ad8ec0846c37e9d0af20d2cdf37022b6a6593e9ec373586","md5":"a15f154a341df8077c7b9d3b6410786e","sha256":"5b0cb7a209b12c8c6bafe6df475ae06d29a2e5f08aa49b5039ea0af235e60080"},"downloads":-1,"filename":"yoto_exception-0.1.1-py3-none-any.whl","has_sig":false,"md5_digest":"a15f154a341df8077c7b9d3b6410786e","packagetype":"bdist_wheel","python_version":"py3","requires_python":">=3.9","size":13301,"upload_time":"2020-11-16T11:30:18","upload_time_iso_8601":"2020-11-16T11:30:18.141193Z","url":"https://files.pythonhosted.org/packages/f5/5e/17e2e180b9f99ad8ec0846c37e9d0af20d2cdf37022b6a6593e9ec373586/yoto_exception-0.1.1-py3-none-any.whl","yanked":false,"yanked_reason":null},{"comment_text":"","digests":{"blake2b_256":"d24bc3320b49fd3273639cbdb979995f7435c64911adc75ae9a8be134956d1f8","md5":"e2fd8b02c33ab8f1522fb2ebca63c392","sha256":"cc78a0ebd02a5eafd3018fffe54ac70148d488738979317e09796a3955e3c4be"},"downloads":-1,"filename":"yoto_exception-0.1.1.tar.gz","has_sig":false,"md5_digest":"e2fd8b02c33ab8f1522fb2ebca63c392","packagetype":"sdist","python_version":"source","requires_python":">=3.9","size":10537,"upload_time":"2020-11-16T11:30:19","upload_time_iso_8601":"2020-11-16T11:30:19.277257Z","url":"https://files.pythonhosted.org/packages/d2/4b/c3320b49fd3273639cbdb979995f7435c64911adc75ae9a8be134956d1f8/yoto_exception-0.1.1.tar.gz","yanked":false,"yanked_reason":null}],"vulnerabilities":[]}}